CMakeLists.txt

```diff
@@ -19,16 +14,37 @@ include( ${PROJECT_NAME}_compiler_flags )
 include(GNUInstallDirs)
 
 ## Dependencies
-find_package( jedicmake QUIET )
 if(OPENMP)
   find_package( OpenMP COMPONENTS Fortran )
 endif()
-find_package( NetCDF REQUIRED COMPONENTS Fortran )
+
```

Comment:
Just a note, with spack-stack lines 20-25 aren't necessary, but then the env. variables `NETCDF` and `NETCDF_PATH` aren't set either, so no problem.

Reply:
I'm aiming for one configuration of CRTM that works for both spack-stack users and non spack-stack users (by far the larger number at this point), so I have to maintain an independent capability to find openMP and netCDF.